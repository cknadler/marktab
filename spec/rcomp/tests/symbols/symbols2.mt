# multiplying sequence symbol
seq: [1:1 2:1 3:1]

seq ^ 2
