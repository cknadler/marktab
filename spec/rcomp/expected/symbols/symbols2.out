e|-1-----1-----|
B|---1-----1---|
G|-----1-----1-|
D|-------------|
A|-------------|
E|-------------|

